### ISSUE TITLE
Email variable placeholders aren't being replaced in TinyMCE input when saving the form

### ISSUE THREAD

**Issue by Vitaliy-1** [pkp/pkp-lib#7689]
**Describe the bug**
When form with email editing input is initialized, email template variables are often replaced with placeholders (if actual data is unknown during email editing), like:
```
<span class="pkpTag mceNonEditable" data-symbolic="passwordResetUrl">URL</span>
``` 
After saving a form, these placeholders aren't replaced with variables, e.g., `{$passwordResetUrl}`. Observing this behaviour only in the main branch and suspect that this is related to the recent migration to TinyMCE 5. 

**To Reproduce**
Steps to reproduce the behavior, e.g., in review reminder form
1. Make a submission, pass it to the review stage and assign a reviewer
2. Edit Response due date
3. Click on Send Reminder button
4. Edit email body
5. Click on Send Reminder and check the actual request's `message` input in the form data through browser dev tools

**What application are you using?**
 OJS main branch

**Additional information**
I don't see anything related in the documentation regarding migration from 4th to 5th version and actual replacement is happening in the `SiteHandler`:  https://github.com/pkp/pkp-lib/blob/996879d1f80058c115274cb5b29377a974a812aa/js/controllers/SiteHandler.js#L317. I mean that event `SaveContent` is fired and content is being replaced but this isn't then passed further to the form data when inspecting through browser devtools.

**Comment by Vitaliy-1** [comment]
Must confess that after inspection of TinyMCE 5 code I haven't noticed any changes in the SaveContent event. Our old JS handlers also work the same as in OJS 3.3. 

But anyway the modification of the `content` through the args passed to the event here: https://github.com/pkp/pkp-lib/blob/996879d1f80058c115274cb5b29377a974a812aa/js/controllers/SiteHandler.js#L324 doesn't have any effects, although direct modification with something like:
```
this.setContent($content.html())
``` 
changes the content of the textarea input.

I can fix a bug with this workaround (plus minor modifications of the function above mentioned) but it doesn't address the source. The conflict between 2 TinyMCE instances might explain this. @NateWr, `FieldRichTextarea.vue` imports `tinymce-vue`, is it possible that it causes the problem?

**Comment by NateWr** [comment]
> I can fix a bug with this workaround (plus minor modifications of the function above mentioned) but it doesn't address the source. 

It seems like your proposed fix would address this. What is the source of the problem that isn't solved here?

**Comment by Vitaliy-1** [comment]
Yes, I found the fix for this particular issue but I haven't found the reason why the value of TinyMCE textarea input doesn't change by modifying its content through the arguments passed to the TinyMCE's `SaveContent` event: https://github.com/tinymce/tinymce/blob/7ca5270dd97019fe756ba68444442bfb47aa0e3b/modules/tinymce/src/core/main/ts/api/Editor.ts#L742. 

I'm worrying that this also may affect other parts of the code executed by handlers after triggering form submission.

**Comment by NateWr** [comment]
Every TinyMCE in the "old" forms should be controlled by the `SiteHandler`. So this should do the trick for everywhere...

**Comment by Vitaliy-1** [comment]
PRs
pkp-lib: https://github.com/pkp/pkp-lib/pull/7789
tests: https://github.com/pkp/ojs/pull/3346

**Comment by NateWr** [comment]
Thanks @Vitaliy-1!
